src/app/api/graphql/route.js

```diff
@@ -1,40 +1,63 @@
-import { startServerAndCreateNextHandler } from '@as-integrations/next'
-import { ApolloServer } from '@apollo/server'
 import { ApolloServerPluginLandingPageLocalDefault, ApolloServerPluginLandingPageProductionDefault } from '@apollo/server/plugin/landingPage/default'
-import { gql } from 'graphql-tag'
+
+import { ApolloServer } from '@apollo/server'
 import { MongoClient } from 'mongodb'
+import { gql } from 'graphql-tag'
+import { startServerAndCreateNextHandler } from '@as-integrations/next'
 
 // The connection string for mongodb connection.
-const uri = process.env.MONGODB_URI || ''
+const uri = process.env.MONGODB_URI || 'mongodb+srv://bacpactech:BACPAC2023@bacpac.e1tsleh.mongodb.net'
```

Comment:
Let's migrate the following keys to an env file. For GitHub we'll need it to add them as tokens. So, we'll have to recheck again if everything stays in sync.